.config/forest.dic

```diff
@@ -33,3 +33,10 @@ tuple
 validator/S
 mempool
 sortition
+deserialize/GS
```

Comment:
We are already using in other place of the codebase the `de-serialize` form. We might want to be orthogonal and use `de-serialize` everywhere (or `deserialize` everywhere).

Reply:
Agreed. Changed to use of de-serialize everywhere